tests/integration/helpers/cluster.py

```diff
@@ -1536,6 +1536,7 @@ def add_instance(
         allow_analyzer=True,
         hostname=None,
         env_variables=None,
+        exclusive_env_variables=False,
```

Comment:
IMO it is better to name it 'instance_env_variables' since those are instance-specific variables.

Reply:
Actually, I think both names are not very good. They give the idea this property is of map type, while it's a boolean. Maybe something like:
1. `set_env_variables_only_to_this_instance`
2. `keep_env_variables_to_this_instance`
3. `make_env_variables_exclusive_to_this_instance`

Or I could "invert" the logic and name it something like: `share_env_variables_within_cluster` and default it to `true`

Reply:
There are perhaps no ideal names, so all that is just matter of preference. Two things here: 
- `instance_env_variables` "links" with the fact that we are adding a new instance and set env variables exclusively to it. To make things less ambiguous `env_variables` must have been `cluster_env_variables`, but
- renaming existing option would affect hundreds of already existing tests and couple of not yet merged ones.